browser/browsing_data/brave_clear_browsing_data.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BROWSING_DATA_BRAVE_CLEAR_BROWSING_DATA_H_
+#define BRAVE_BROWSER_BROWSING_DATA_BRAVE_CLEAR_BROWSING_DATA_H_
+
+namespace content {
+class BrowsingDataRemover;
+}
+class Profile;
+class BraveClearDataOnExitTest;
+
+namespace content {
+
+class BraveClearBrowsingData {
```

Comment:
not sure if we need this class at all, I think free functions will work as well

Reply:
Could do it with functions, but I like the grouping the the class provides.